CRM/Core/DAO/ActionSchedule.php

```diff
@@ -37,6 +37,17 @@ class CRM_Core_DAO_ActionSchedule extends CRM_Core_DAO {
    */
   public static $_log = FALSE;
 
+  /**
+   * Paths for accessing this entity in the UI.
+   *
+   * @var string[]
+   */
+  protected static $_paths = [
+    'add' => 'civicrm/admin/scheduleReminders/edit?reset=1&action=add',
```

Comment:
If you look at the main listing page (`/civicrm/admin/scheduleReminders?reset=1`), the `Add Reminder" button still goes to the old URL (e.g. `http://core-26884-62nfn.test-1.civicrm.org:8001/civicrm/admin/scheduleReminders?action=add&reset=1`). Shouldn't that change to match?